src/bun.js/api/bun/subprocess.zig

```diff
@@ -2319,8 +2326,12 @@ pub const Subprocess = struct {
     pub fn handleIPCClose(this: *Subprocess) void {
         IPClog("Subprocess#handleIPCClose", .{});
         this.updateHasPendingActivity();
-        const ok = this.ipc_data != null;
-        if (ok) this.ipc().internal_msg_queue.deinit();
+        defer this.deref();
```

Comment:
only if `ipc_data` is not null right?

Reply:
we always need to deref here because always we ref earlier